### PR TITLE
ENH: Add Python code for kernel creation examples

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add Python code for kernel creation examples.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Have not tested locally.

**Comment by jhlegarreta** [comment]
Pinging @mseng10

File: src/Core/Common/CreateDerivativeKernel/Documentation.rst [hunk 1]
```diff
@@ -39,6 +39,14 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateDerivativeKernel/Documentation.rst:47]: `18-` to `16-`

File: src/Core/Common/CreateForwardDifferenceKernel/Documentation.rst [hunk 2]
```diff
@@ -39,6 +39,14 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateForwardDifferenceKernel/Documentation.rst:47]: `18-` to `16-`

File: src/Core/Common/CreateGaussianDerivativeKernel/Documentation.rst [hunk 3]
```diff
@@ -39,6 +39,14 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateGaussianDerivativeKernel/Documentation.rst:47]: `18-` to `16-`

File: src/Core/Common/CreateGaussianKernel/Documentation.rst [hunk 4]
```diff
@@ -40,6 +40,13 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateGaussianKernel/Documentation.rst:48]: `18-` to `16-`

File: src/Core/Common/CreateLaplacianKernel/Documentation.rst [hunk 5]
```diff
@@ -39,6 +39,14 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateLaplacianKernel/Documentation.rst:47]: `18-` to `16-`

File: src/Core/Common/CreateSobelKernel/Documentation.rst [hunk 6]
```diff
@@ -38,6 +38,14 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/CreateSobelKernel/Documentation.rst:46]: `18-` to `16-`

Comment (jhlegarreta) [reply src/Core/Common/CreateSobelKernel/Documentation.rst]: Done in be2ebf0.

File: src/Core/Common/CreateLaplacianKernel/Documentation.rst (hunk 5 above)
Comment (jhlegarreta) [reply src/Core/Common/CreateLaplacianKernel/Documentation.rst]: Done in be2ebf0.

File: src/Core/Common/CreateGaussianKernel/Documentation.rst (hunk 4 above)
Comment (jhlegarreta) [reply src/Core/Common/CreateGaussianKernel/Documentation.rst]: Done in be2ebf0.

File: src/Core/Common/CreateGaussianDerivativeKernel/Documentation.rst (hunk 3 above)
Comment (jhlegarreta) [reply src/Core/Common/CreateGaussianDerivativeKernel/Documentation.rst]: Done in be2ebf0.

File: src/Core/Common/CreateForwardDifferenceKernel/Documentation.rst (hunk 2 above)
Comment (jhlegarreta) [reply src/Core/Common/CreateForwardDifferenceKernel/Documentation.rst]: Done in be2ebf0.

File: src/Core/Common/CreateDerivativeKernel/Documentation.rst (hunk 1 above)
Comment (jhlegarreta) [reply src/Core/Common/CreateDerivativeKernel/Documentation.rst]: Done in be2ebf0.

**Pull request merged**